Resources/Prototypes/GameRules/roundstart.yml

```diff
@@ -142,6 +142,33 @@
     startingChaosRatio: 0.025 # Starts as slow as survival, but quickly ramps up
     shiftLengthModifier: 2.5
 
+- type: entity
+  id: IrregularStationEventScheduler
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: OscillatingStationEventScheduler
+    minChaos: 0.8
+    maxChaos: 14
+    startingSlope: 0.2
+    downwardsLimit: -0.35
+    upwardsLimit: 0.4
+
+# More likely to go down than up, so calmness prevails
+- type: entity
+  id: IrregularExtendedStationEventScheduler
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: OscillatingStationEventScheduler
```

Comment:
This should have comments on every line to explain what each component does, just to save whoever comes looking from finding the comp and reading the variables there.

Reply:
Reading the documentation is exactly what you're supposed to do when introducing changes. Each of those fields has a docstring which explain what it does.